frontend/src/pages/Notifications/NotificationList.js

```diff
@@ -71,15 +71,19 @@ const NotificationList = props => {
         </Button>
       </div>
 
-      <List component="div" data-test="notification-list">
-        <NotificationListItems
-          notifications={notifications}
-          history={history}
-          markNotificationAsRead={notificationId => markNotificationAsRead(notificationId, currentPage)}
-          notificationsPerPage={notificationsPerPage}
-          notificationOffset={notificationOffset}
-        />
-      </List>
+      {isDataLoading ? (
+        <div />
+      ) : (
```

Comment:
the check if the "empty state" or the notification list is displayed should be done here to make it more explicit, i.e. _either_ we display the empty state _or_ the list of notifications

Reply:
we could even put the check one layer above in the `NotificationPage`